src/encord_active/frontend/src/components/explorer/display/Display.tsx

```diff
@@ -0,0 +1,132 @@
+import { Button, Select, Slider, Space, Switch } from "antd";
+import { Dispatch, SetStateAction } from "react";
+import { TbSortAscending, TbSortDescending } from "react-icons/tb";
+import { MinusSquareFilled, TableOutlined } from "@ant-design/icons";
+import { Metric } from "../ExplorerTypes";
+import { ProjectDomainSummary } from "../../../openapi/api";
+import { FilterState, updateValue } from "../../util/MetricFilter";
+
+type Props = {
+  selectedMetric: Metric;
+  predictionHash: string | undefined;
+  dataMetricsSummary: ProjectDomainSummary;
+  annotationMetricsSummary: ProjectDomainSummary;
+  isSortedByMetric: boolean;
+  setSelectedMetric: (val: Metric) => void;
+  isAscending: boolean;
+  setIsAscending: Dispatch<SetStateAction<boolean>>;
+  showAnnotations: boolean;
+  toggleShowAnnotations: any;
+  setConfidenceFilter: (
+    arg: FilterState | ((old: FilterState) => FilterState)
+  ) => void;
+  gridCount: number;
+  setGridCount: (val: number) => void;
+};
+export function Display({
+  predictionHash,
+  dataMetricsSummary,
+  annotationMetricsSummary,
+  isSortedByMetric,
+  selectedMetric,
+  setSelectedMetric,
+  isAscending,
+  setIsAscending,
+  showAnnotations,
+  toggleShowAnnotations,
+  setConfidenceFilter,
+  gridCount,
+  setGridCount,
+}: Props) {
+  return (
+    <div className="flex flex-col gap-2 divide-y ">
+      <div className="icon-wrapper flex w-full items-center gap-2">
+        <MinusSquareFilled className="text-base text-gray-8" />
+        <Slider
+          min={1}
+          max={12}
+          defaultValue={gridCount ?? 4}
+          className="w-full"
+          trackStyle={{
+            backgroundColor: "#434343",
+          }}
+          onChange={(val) => setGridCount(val)}
+        />
+        <TableOutlined className="text-base text-gray-8" />
+      </div>
+      <div className="flex items-center justify-between">
+        <div className="px-2 py-4 text-sm text-gray-8">Display Labels</div>
+        <Switch
+          onClick={toggleShowAnnotations}
+          className="bg-gray-9"
+          checked={!showAnnotations}
+        />
+      </div>
+      <div className="flex items-center justify-between">
+        <div className="px-2 py-4 text-sm text-gray-8">Display Predictions</div>
+        <Switch disabled className="bg-gray-9" />
+      </div>
+      <div className="flex flex-col items-start justify-between p-4">
+        <div className=" text-sm text-gray-8">Confidence</div>
+        <Slider
+          range
+          defaultValue={[0, 1]}
+          min={0}
+          max={1}
+          step={0.01}
+          className="w-full"
+          onAfterChange={(newRange: [number, number]) => {
+            setConfidenceFilter(
```

Comment:
May want to keep confidence as just a standard metric filter for now as we don't support it in the TP/FP/FN condition - so special case logic may confuse the user.

Also - this can get out of sync with a confidence filter added manually to the standard list of metric filters (though i think that errors).

Reply:
Fixed, Made confidence bounds and values populate from FIlterState for annotation filters.